namerd/README.md

```diff
@@ -39,3 +39,47 @@ or
 ```
 ./sbt 'namerd-main:run path/to/config.yml'
 ```
+
+## Http Control Interface ##
+
+namerd provides an http interface to create, update and retrieve Dtabs, as
+defined in `HttpControlService`.
+
+For a command-line tool which wraps this interface, try
+[namerctl](https://github.com/BuoyantIO/namerctl)
+
+Primary interface endpoints:
+
+```
+$ export NAMERD_URL=http://localhost:4180
+```
+
+Get all dtab namespaces:
+
+```
+$ curl $NAMERD_URL/api/1/dtabs
+```
+
+Get a dtab namespace:
```

Comment:
tioli: it might be clearer to say:
Get a dtab for the `default` namespace
since default is a common word...